huxley/www/js/components/ChairSummaryView.js

```diff
@@ -188,8 +192,9 @@ var ChairSummaryView = React.createClass({
   _handleSummaryChange(assignment, event) {
     var newSummary = event.target.value;
     var summaries = this.state.summaries;
-    summaries[assignment.id] = newSummary;
-    this.setState({summaries: summaries});
+    var update = {};
+    update[assignment.id] = newSummary;
+    this.setState({summaries: {...summaries, ...update}});
```

Comment:
Don't need an update object here, if you used computed property syntax (which I'm pretty sure we have the transform for):
```js
this.setState({
  summaries: {
    ...this.state.summaries,
    [assignment.id]: event.target.value,
  },
});
```